src/hooks/useAnalytics.ts

```diff
@@ -104,7 +104,9 @@ export const useAnalytics = ({
   venue: space,
 }: UseAnalyticsOptions): UseAnalyticsResult => {
   const { user } = useLoginCheck();
-  const { world, isLoaded: isWorldLoaded } = useWorldById(space?.worldId);
+  const { world, isLoaded: isWorldLoaded } = useWorldById({
+    worldId: space?.worldId,
+  });
```

Comment:
[Q] Does this make TS complain?

```suggestion
  const { world, isLoaded: isWorldLoaded } = useWorldById(space);
```